src/components/Security/Users/Steps/UserProfileList.vue

```diff
@@ -86,6 +91,7 @@ export default {
     },
     onProfileSelected(profile) {
       this.$emit('selected-profile', profile)
+      this.selectedProfiled = 0
```

Comment:
When adding the last profile of the list, there is no more profiles to add so the selectedProfile changed and that trigger this function and add to the two level parent an undefined profile badge

```suggestion
    onProfileSelected(profile) {
      if (!profile) {
        return;
      }
      this.$emit('selected-profile', profile)
      this.selectedProfiled = 0
```